ci/ci_test.sh

```diff
@@ -7,6 +7,8 @@
 
 SELECTED_TEST_SUITE=$1
 
+export SPEC_OPTS="--order rand --format documentation --format CI::Reporter::RSpecFormatter"
```

Comment:
documentation format: its the only way to know where a test "stalls" if it stalls. with dots you only see dots and there's no way to know which test has stalled.

documentation option: my thinking is that it makes it more explicit and not rely on any default value.